.github/workflows/vm_full_tests.yml

```diff
@@ -0,0 +1,60 @@
+name: AVM Full Tests
+
+on:
+  schedule:
+    - cron: "15 4 * * 1-5" # Monday to Friday at 4:15 AM UTC
+  workflow_dispatch:
+    inputs:
+      username:
+        description: "Username (optional)"
+        required: false
+      action:
+        description: "Default to 'start'"
+        required: false
+        default: "start"
+
+concurrency:
+  # force parallelism in master
+  group: avm-full-tests-${{ github.ref }}
+  cancel-in-progress: true
+
+env:
+  DOCKERHUB_PASSWORD: "${{ secrets.DOCKERHUB_PASSWORD }}"
+  RUN_ID: ${{ github.run_id }}
+  RUN_ATTEMPT: ${{ github.run_attempt }}
+  USERNAME: ${{ github.event.pull_request.user.login || github.actor }}
+  GITHUB_TOKEN: ${{ github.token }}
+  GH_SELF_HOSTED_RUNNER_TOKEN: ${{ secrets.GH_SELF_HOSTED_RUNNER_TOKEN }}
+  AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+  AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+  BUILD_INSTANCE_SSH_KEY: ${{ secrets.BUILD_INSTANCE_SSH_KEY }}
+  GIT_COMMIT: ${{ github.event.pull_request.head.sha || github.sha }}
+  # kludge until we move away from runners
+  WAIT_FOR_RUNNERS: false
+
+jobs:
+  setup:
+    uses: ./.github/workflows/setup-runner.yml
+    with:
+      username: ${{ github.event.pull_request.user.login || github.actor }}
```

Comment:
whose runner do we want this to run on? I guess right now it'd be github bot

Reply:
@ludamad  Ok good point. How should I change this exactly?

Should I also change the argument in "runs-on" at line 47?

Reply:
Everywhere we consider the GitHub actor. You can see our e2e CI file as an example. We want to start the runner with the same flags as our runs on

Reply:
Sorry, the arm Ci as an example

Reply:
Thx for the pointer. I followed the arm CI template.

Reply:
@ludamad I had to revert the changes made after your feedback. Somehow the workflow did not leverage cache for build and I had 2 hours build for wasm and MacOSX. I tried to remove wasm dependencies in Earthfile without success. Therefore, I am going to merge this version.